packages/cost/models/authors/moonshotai/kimi-k2/endpoints.ts

```diff
@@ -3,50 +3,6 @@ import type { ModelProviderConfig } from "../../../types";
 import { KimiK2ModelName } from "./models";
 
 export const endpoints = {
-  "kimi-k2:groq": {
```

Comment:
Groq deprecated this model and is now routing every call to this model to their 0905 counterpart : https://console.groq.com/docs/model/moonshotai/kimi-k2-instruct